portfolio/src/main/webapp/scripts/login.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Shows feedback on login status
+ */
+async function showCommentFeedback() {
+  try {
+    const response = await fetch('/create-new-comment');
+    const loginStatus = await response.text();
+    document.getElementById('login-feedback').innerText = loginStatus;
+  } catch {
+    alert('Failed to fetch login status.'); 
+  }
+
+}
+
+/**
+ * Changes button text to reflect whether user should log in or out
+ */
+async function updateLoginStatusOnButton() {
+  try {
+    const response = await fetch('/check-login-status');
+    const loginStatus = await response.text();
+    document.getElementById('login-button').innerText = loginStatus;
+  } catch {
+    document.getElementById('login-button').innerText = "Log in/out"; 
```

Comment:
While I get that this is an error state, "log in/out" is a very confusing message to a user.

Reply:
I have updated the button's text content